pkgs/development/web/nodejs/nodejs.nix

```
@@ -130,6 +134,35 @@ let
 
       # install the missing headers for node-gyp
       cp -r ${concatStringsSep " " copyLibHeaders} $out/include/node
+
+      # assemble a static v8 library and put it in the 'libv8' output
+      mkdir -p $libv8/lib
+      pushd out/Release/obj.target
+      find . -path "./torque_*/**/*.o" -or -path "./v8*/**/*.o" | sort | uniq >files
```

```suggestion
      find . -path "./torque_*/**/*.o" -or -path "./v8*/**/*.o" | sort -u >files
```
sort can do that and maybe it is even faster for a lot of files.

---

Yeah, good point. It's probably a bit faster.